configs/webmidi.json

```
@@ -1,27 +1,52 @@
 {
   "index_name": "webmidi",
-  "start_urls": [
-    "https://djipco.github.io/webmidi/"
-  ],
+  
   "sitemap_urls": [
     "https://djipco.github.io/webmidi/sitemap.xml"
   ],
   "sitemap_alternate_links": true,
   "stop_urls": [],
+  
+  "start_urls": [
+    "https://djipco.github.io/webmidi/",
+    {
+      "url": "https://djipco.github.io/webmidi/api/",
+      "selectors_key": "docs"
+    },
+    {
+      "url": "https://djipco.github.io/webmidi/docs/",
+      "selectors_key": "docs"
+    }
```

```suggestion
    {
      "url": "https://djipco.github.io/webmidi/api/",
      "selectors_key": "docs"
    },
    {
      "url": "https://djipco.github.io/webmidi/docs/",
      "selectors_key": "docs"
    },
    "https://djipco.github.io/webmidi/"
```

deeper path need to first otherwise they broader one will match for all URLs